clang/unittests/Format/FormatTest.cpp

```diff
@@ -16339,7 +16345,7 @@ TEST_F(FormatTest, ConfigurableSpaceBeforeParens) {
 
   verifyFormat("int f();", SpaceFuncDef);
   verifyFormat("void f (int a, T b) {}", SpaceFuncDef);
-  verifyFormat("A::A () : a(1) {}", SpaceFuncDef);
+  verifyFormat("A::A() : a(1) {}", SpaceFuncDef);
```

Comment:
But that is a function definition, is it not?

Reply:
The space was inserted in 086305120887 because ctors/dtors were annotated as `FunctionDeclarationName`. Now that we annotate them as `CtorDtorDeclName`, they are no longer functions.